pandas/tests/indexing/test_scalar.py

```diff
@@ -302,3 +303,29 @@ def test_iat_dont_wrap_object_datetimelike():
         assert result is ser2[1]
         assert isinstance(result, timedelta)
         assert not isinstance(result, Timedelta)
+
+
+def test_multiindex_series_loc_get():
+    series = pd.Series([1, 2], index=[[1, 2], [3, 4]])
+    assert series.loc[1, 3] == 1
+
+
+def test_multiindex_series_loc_set():
+    series = pd.Series([1, 2], index=[[1, 2], [3, 4]])
+    series.loc[1, 3] = 3
+    assert series.loc[1, 3] == 3
```

Comment:
i dont think the loc tests are needed, unless that behavior is being fixed

Reply:
I believe @jreback requested loc to be added because the rest of the tests found in test_scalar test both at and loc.